docs/cow-protocol/tutorials/cow-amm-deployer.mdx

```diff
@@ -2,82 +2,43 @@
 sidebar_position: 6
 ---
 
-# CoW AMM Deployer
-The [CoW AMM Deployer](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fdeploy-cow-amm.bleu.fi&chain=et) is a user-friendly Safe App that simplifies the process of deploying and managing [CoW AMMs](https://cow.fi/cow-amm) (Automated Market Makers) from Safe wallets. It enables users without programming skills to take advantage of the benefits of CoW AMM. 
+# Initial notes
 
-The app was developed by [bleu](https://github.com/bleu-fi) with sponsorship from the [CoW Grants DAO](https://grants.cow.fi/).
+The proposed transactions and interactions, can be executed through Safe's transaction builder, etherscan "Write Contract" or from other UI mechanisms or directly from console.
 
-This app is only accessible from a Safe wallet. 
+# Deploying a CoW AMM
 
-:::caution
+In this short tutorial, we describe how one can create and deploy their own Balancer CoW AMM.
+
+The current factory contract addresses are the following:
+- Ethereum Mainnet: [`0x23fcC2166F991B8946D195de53745E1b804C91B7`](https://etherscan.io/address/0x23fcC2166F991B8946D195de53745E1b804C91B7#code)
+- Gnosis Chain: [`0x7573B99BC09c11Dc0427fb9c6662bc603E008304`](https://gnosisscan.io/address/0x7573B99BC09c11Dc0427fb9c6662bc603E008304#code)
+- Sepolia Testnet: [`0x9F151748595bAA8829d44448Bb3181AD6b995E8e`](https://sepolia.etherscan.io/address/0x9F151748595bAA8829d44448Bb3181AD6b995E8e#code)
 
-The Safe wallet you use to create a CoW AMM should **ONLY** be used for the purpose of deploying a CoW AMM.
+1. In order to create a new pool, one first needs to call the `newBPool` function of the factory contract. An example transaction that creates a new pool can be found [here](https://etherscan.io/tx/0x7543a97853827e267ecd3c1309509ac7704e4f85a53fbfacd6060f461d85bad8#eventlog). The relevant `LOG_NEW_POOL` log also reveals the address of the newly created pool; in our example transaction, it is `0x81530e9B069c69F6671A0A9d7Ee337cafEF419F6`.
 
+:::note
+
+In case ABI of the factory contract is not fetched, the ABI from the Sepolia network can be used.
 :::
 
-:::tip
+2. Once the new pool has been created, the next step is to approve the tokens we want to add to the pool; this can be done by calling the `approve()` function of the contract of the relevant ERC-20 token we are interested in adding to the pool, where the "spender" needs to be set to the newly created address.
```

Comment:
```suggestion
2. The next step is to approve the tokens to add to the pool; this can be done by calling the `approve()` function on the contract of the relevant ERC-20 token that is being added to the pool, where the "spender" needs to be set to the newly created BPool address from (1).
```